fake_fire/init.lua

```diff
@@ -224,25 +226,41 @@ minetest.register_node("fake_fire:embers", {
 		aspect_w=16, aspect_h=16, length=2}},
 	},
 	light_source = 9,
-	groups = {crumbly=3},
+	groups = {crumbly=3, dig_stone=2},
 	paramtype = "light",
-	sounds = default.node_sound_dirt_defaults(),
+	_sound_def = {
+		key = "node_sound_dirt_defaults",
+	},
 })
 
+local sandstone_tex = "default_sandstone.png"
+if not minetest.get_modpath("default") then
+	local sname = minetest.registered_nodes["mapgen_stone"].name
```

Comment:
```
2022-05-01 18:26:12: ERROR[Main]: ModError: Failed to load and run script from /home/flux/.minetest/worlds/dev_test/worldmods/homedecor
_modpack/fake_fire/init.lua:
2022-05-01 18:26:12: ERROR[Main]: .../dev_test/worldmods/homedecor_modpack/fake_fire/init.lua:238: attempt to index field 'mapgen_stone
' (a nil value)
2022-05-01 18:26:12: ERROR[Main]: stack traceback:
2022-05-01 18:26:12: ERROR[Main]:       .../dev_test/worldmods/homedecor_modpack/fake_fire/init.lua:238: in main chunk
```

Reply:
seems minetest mod loader isnt loading whatever adds mapgen_stone alias before this. can be fixed with a optional depend on that mod

Reply:
my fault for not adding essential aliases.

however, it can still crash if the node name doesn't have a ':'

```
2022-05-01 18:35:43: ERROR[Main]: ModError: Failed to load and run script from /home/flux/.minetest/worlds/dev_test/worldmods/homedecor_modpack/fake_fire/init.lua:
2022-05-01 18:35:43: ERROR[Main]: .../dev_test/worldmods/homedecor_modpack/fake_fire/init.lua:240: bad argument #1 to 'gsub' (string expected, got nil)
2022-05-01 18:35:43: ERROR[Main]: stack traceback:
2022-05-01 18:35:43: ERROR[Main]:       [C]: in function 'gsub'
2022-05-01 18:35:43: ERROR[Main]:       .../dev_test/worldmods/homedecor_modpack/fake_fire/init.lua:240: in main chunk
```

Reply:
did find another bug, due to devtest breaking right to a name. fixed https://github.com/mt-mods/homedecor_modpack/pull/28/commits/d5301e1eca6f11104b808e675eefcad346245cb9